src/templates/typescript/interfaces/interfaceTemplate.ts

```diff
@@ -0,0 +1,31 @@
+import MemberVariable from "../../../model/dataTypes/MemberVariable";
+import Handlebars from "../registerPartials";
+import DataType from "../../../model/dataTypes/DataType";
+import EntityMetadata from "../../../model/dataTypes/EntityMetadata";
+import EntityImports from "../../../model/dataTypes/EntityImports";
+
+export const source = `{{> interfaceImport dataType=this }}
+export default interface I{{ dataType.normalizedName }}{{# if parentDataType }} extends I{{ parentDataType.normalizedName }}{{/ if }} {
+{{# if memberVariables }}
+  {{# each memberVariables }}
+  {{# if dataType.primitive }}
+  {{> primitiveMember member=this }}
+  
+  {{ else }}
+  {{> interfaceMember member=this }}
+  
+  {{/ if }}
+  {{/ each }}
+{{/ if }}
+}
+`;
+
+export interface ITemplateContext {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)